nntrainer/graph/network_graph.cpp

```diff
@@ -85,14 +85,26 @@ int NetworkGraph::compile(const std::string &loss_type) {
 
 void NetworkGraph::setExecutionOrder() {
   auto max_count = graph.size() * 3;
-  /** @todo: remove backwarding count for non-trainble layers */
+  size_t not_trainables = 0;
```

Comment:
I think, We need to check if the max_count is the same with ```std::get<2>(backward_iter_end->getExecutionOrder()``` . Because the max is used at ```tensor_manager->allocateTensors(std::get<2>(backward_iter_end->getExecutionOrder())```

Reply:
it's changed to iterate reversely with @lhs8928 's comment.  please check.